src/components/ZoomEffect.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import * as styles from "./ZoomEffect.module.css";
+import classNames from "classnames";
+
+const ZoomEffect = ({ children, in: inProp }) => {
+    return (
+        <div className={classNames(styles.zoomTransition,{
+            [styles.zoomEnter]: inProp === true,
+            [styles.zoomExit]: inProp === false,
+        })}>
+            {children}
+        </div>
+    );
+};
+
+export default ZoomEffect;
```

Comment:
Could you add the final line to this file, please?

![image](https://github.com/user-attachments/assets/29d914cf-ffc2-4501-af9f-e0b2a1790e25)

Reply:
Done!